modules/ROOT/pages/apid-behavioral-headers.adoc

```diff
@@ -34,16 +30,48 @@ Examples:
 }
 ----
 
-MS2-Error-Level:: Indicates which range of error status codes the mocking service is restricted to using. You can specify one of the following three case-sensitive values:
-+
+== MS2-Example
+
+The MS2-Example header enables you to choose a specific response example if your endpoint defines more than one example for the same status code.
+
+Using MS2-Example, you can specify multiple examples within a response:
+
+----
+/examples:
+  get:
+    responses:
+      200:
+        body:
+          examples:
+            one:
+              prop1: something
+              prop2: 30
+            two:
+              prop1: another
+              prop2: 31
+            three:
+              prop1: anything
+              prop2: 33
+----
+
+Here, the possible values of the header would be `one`, `two`, and `three`. Sending the value `one` as part of the request would return the properties `something` and `30`. Sending a value other than `one`, `two`, and `three` would return no results.
```

Comment:
Change the word properties to payload.
Also, change: would return no results. to would return an error message.

Reply:
Updated.